### DISCUSSION TITLE
How should "account required" cases be handled? Will WARC help?

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uAssets#16088] [Ideas]
I oftentimes find breakage problems but behind a page that requires account or it was a ad-shortener link, making reproduction difficult.
How should we approach this? Passing account login information, or supplying a WARC?

**Comment by stephenhawk8054** [comment]
What is a `WARC`?

**Reply by okiehsch** [reply]
> How should we approach this?

It's really up to you, if it's not a sensitive site, i.e. shopping or banking, you can privately share your login information, or especially if the issue is about site breakage you can provide your logger output of the blocked requests on your end and we can try to debug it that way.
